@factor/@themes/theme-ultra/src/page-home.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div id="homeContainerID" class="homeContainer">
```

Comment:
We try and steer clear of using the ID attribute.

Reply:
In my next PR I will change that, thanks!